### ISSUE TITLE
provider: `AKASH_CLUSTER_WAIT_READY_DURATION` code has been removed in akash `v0.16.5-rc0`

### ISSUE THREAD

**Issue by andy108369** [akash-network/support#7]
There is a flag `AKASH_CLUSTER_WAIT_READY_DURATION` which we are setting to `30s` in helm-chart based provider:

```
node$ provider-services run --help | grep ready
      --cluster-wait-ready-duration duration        The time to wait for the cluster to be available (default 5s)
```

Trying to figure out the difference between Praetor-based and helm-chart based provider install, I've only noticed `AKASH_CLUSTER_WAIT_READY_DURATION` is not being set on the former.
As I've suggested Praetor team to try setting this parameter to `60s` to rule out the [issue](https://github.com/ovrclk/engineering/issues/728#issuecomment-1399564645) where the leases would get removed upon provider (systemd service in their case) restart, I looked into the code and haven't found any reference this flag is used...

The last reference I found is the one which was removed back in Akash `v0.16.5-rc0` according to this commit:
https://github.com/akash-network/node/commit/f161334fec87a0c4b072077a9147d6cd21138427#diff-d9bf78faae6e633425de7ee74e6d6f7a1ffffebdb2e4e62bbef7f759d5783594L84-L93  (look for `cfg.ClusterWaitReadyDuration` in `provider/service.go`)

Do you think it should be put back?
If not, then this flag should also be gone.

**Comment by troian** [comment]
it is still located https://github.com/akash-network/provider/blob/76acbe7736f8495fb8dd8f20b5c075395f42f455/cmd/provider-services/cmd/run.go#L610
and both env variables `AKASH_CLUSTER_WAIT_READY_DURATION` `AP_CLUSTER_WAIT_READY_DURATION` or flag`cluster-wait-ready-duration` should work

**Comment by andy108369** [comment]
> it is still located https://github.com/akash-network/provider/blob/76acbe7736f8495fb8dd8f20b5c075395f42f455/cmd/provider-services/cmd/run.go#L610 and both env variables `AKASH_CLUSTER_WAIT_READY_DURATION` `AP_CLUSTER_WAIT_READY_DURATION` or flag`cluster-wait-ready-duration` should work

Yeah, they look like leftovers to me... Are they really used?

**Comment by troian** [comment]
yes, tho people should use what is default value

**Comment by andy108369** [comment]
> yes, tho people should use what is default value

I agree. It makes me wonder why it was increased at all... Steven probably did that back then.

I'll drop it in the helm-charts provider so it gets to the default value.
